Integration tests failing

The integration Tests (POSIX) are failing:

```
Run make integration-tests
docker-compose up -d
make: docker-compose: No such file or directory
make: *** [Makefile:107: .idp.docker] Error [12](https://github.com/techservicesillinois/awscli-login/actions/runs/10532232313/job/29185995020#step:13:13)7
Error: Process completed with exit code 2.
```

--- a/Makefile
+++ b/Makefile
@@ -104,11 +104,11 @@ test: lint static check .coverage
 
 idp: .idp.docker
 .idp.docker:
-	docker-compose up -d
+	docker compose up -d
 	@touch $@
 
 idp-down:
-	docker-compose down --rmi all
+	docker compose down --rmi all
 	rm -f .idp.docker
 
 .install-build: $(RELEASE)

